packages/nimble-components/src/table/template.ts

```diff
@@ -37,7 +37,7 @@ export const template = html<Table>`
             --ni-private-table-scroll-height: ${x => x.virtualizer.allRowsHeight}px;
             --ni-private-table-row-container-top: ${x => x.virtualizer.rowContainerYOffset}px;
             --ni-private-table-row-grid-columns: ${x => x.rowGridColumns ?? ''};
-            ">
+        ">
```

Comment:
FYI it looks like you lost the line setting the `disable-select` style in your last merge from main, before that the class attribute was
`class="${x => (x.documentShiftKeyDown ? 'disable-select table-container' : 'table-container')}"`

Reply:
Ah, thanks for pointing that out. There were a lot of merge conflicts that I had to resolve, and I must have missed it.